boost/utils.py

```diff
@@ -778,4 +778,4 @@ def recognize(self, t):
 options = {'hide_intrusive_hooks': True}
 
 # Latest boost currently supported by printers
-last_supported_boost_version = (1, 73, 0)
+last_supported_boost_version = (1, 74, 0)
```

Comment:
Please exclude that from MR. Updating boost version has to me done separately, after version release.